tests/unit/scripting.tcl

```diff
@@ -113,6 +113,11 @@ start_server {tags {"scripting"}} {
         run_script {return redis.call('get',KEYS[1])} 1 mykey
     } {myval}
 
+    test {EVAL - is Lua able to call Server API?} { 
+        r set mykey myval
+        run_script {return server.call('get',KEYS[1])} 1 mykey 
+    }
```

Comment:
I think we can some fancy stuff in run_script to find replace all instances of `redis.` with `server.` to get very comprehensive coverage. For now, I think this is okay. We can write a more comprehensive test later.

Reply:
yeah, I started watching TCL tutorials after I saw 193 instances of redis.call in this file. We are short on time for me to investigate that right now, so added sanity test only.